{"uid":"1c2466716f5388c2","name":"Login with invalid credentials -- @1.1 ","fullName":"Login Functionality: Login with invalid credentials","historyId":"89cf40c23fe57ef543ab1a5ea337cab6","time":{"start":1731712670761,"stop":1731712671341,"duration":580},"status":"broken","statusMessage":"KeyError: ''\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/login_steps.py\", line 57, in assert_error_message\n    expected_error_text = Utils.get_localized_text(\"invalid_credentials_error\")\n                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/lego_code_challenge/lego_code_challenge/utilities/utils.py\", line 25, in get_localized_text\n    return translations[locale][key]\n           ~~~~~~~~~~~~^^^^^^^^\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"KeyError: ''\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/login_steps.py\", line 57, in assert_error_message\n    expected_error_text = Utils.get_localized_text(\"invalid_credentials_error\")\n                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/lego_code_challenge/lego_code_challenge/utilities/utils.py\", line 25, in get_localized_text\n    return translations[locale][key]\n           ~~~~~~~~~~~~^^^^^^^^\n","steps":[{"name":"Given the user is on the login page","time":{"start":1731712671192,"stop":1731712671192,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"When the user logs in with invalid: username","time":{"start":1731712671192,"stop":1731712671281,"duration":89},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"Then invalid credentials error message is displayed","time":{"start":1731712671281,"stop":1731712671284,"duration":3},"status":"broken","statusMessage":"KeyError: ''\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/login_steps.py\", line 57, in assert_error_message\n    expected_error_text = Utils.get_localized_text(\"invalid_credentials_error\")\n                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/lego_code_challenge/lego_code_challenge/utilities/utils.py\", line 25, in get_localized_text\n    return translations[locale][key]\n           ~~~~~~~~~~~~^^^^^^^^\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":3,"attachmentsCount":0,"hasContent":true},"afterStages":[],"labels":[{"name":"severity","value":"normal"},{"name":"tag","value":"login"},{"name":"feature","value":"Login Functionality"},{"name":"framework","value":"behave"},{"name":"language","value":"cpython3"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"invalid_field","value":"username"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":2,"skipped":0,"passed":2,"unknown":0,"total":4},"items":[{"uid":"96cc6fb432a6d7dc","reportUrl":"https://andrey-metelsky.github.io/lego_code_challenge/2//#testresult/96cc6fb432a6d7dc","status":"broken","statusDetails":"KeyError: ''\n","time":{"start":1731626327659,"stop":1731626328296,"duration":637}},{"uid":"1a0e0dc7b65101cc","reportUrl":"https://andrey-metelsky.github.io/lego_code_challenge/1//#testresult/1a0e0dc7b65101cc","status":"passed","time":{"start":1731592992767,"stop":1731592993406,"duration":639}},{"uid":"ed00377ff898061","reportUrl":"https://andrey-metelsky.github.io/lego_code_challenge/1//#testresult/ed00377ff898061","status":"passed","time":{"start":1731592639743,"stop":1731592640302,"duration":559}}]},"tags":["login"]},"source":"1c2466716f5388c2.json","parameterValues":["username"]}